### PR TITLE
fix bug in boilerplate

### PR DESCRIPTION
**Pull request by PatWie**
Change FakeData to support different data types and ranges of values. Motivation is to enable ci-tests for the boilerplate template.

### CODE REVIEW

**Comment by ppwwyyxx** [comment]
Why will the domain be useful?

**Comment by PatWie** [comment]
You need the `domain` for fake-labels of type uint8.

File: tensorpack/dataflow/raw.py [hunk 1]
```diff
@@ -14,32 +15,41 @@
...
     def size(self):
         return self._size
 
     def get_data(self):
         if self.random:
             for _ in range(self._size):
-                yield [self.rng.rand(*k).astype(self.dtype) for k in self.shapes]
+                val = []
+                for k in range(len(self.shapes)):
+                    v = self.rng.rand(*self.shapes[k]) * self.domain[k][1] + self.domain[k][0]
```
Comment (ppwwyyxx) [inline tensorpack/dataflow/raw.py]: Domain means (min, range) ? Would it be simpler to use (min, max)?

**Comment by PatWie** [comment]
You are right!

**Pull request merged**